### PR TITLE
[STG71]Edit HISTORY.md and _version

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 12.1.1 (Unreleased)
+## 12.2.0
+**New features**
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-blob/HISTORY.md]: Missing a new line.

File: sdk/storage/azure-storage-blob/HISTORY.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+- Added support for encryption scopes that would be used to encrypt the application data.
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-blob/HISTORY.md]: maybe "that could be used to encrypt blob content"?

File: sdk/storage/azure-storage-blob/HISTORY.md (hunk 1 above)
Comment (zezha-msft) [reply sdk/storage/azure-storage-blob/HISTORY.md]: Add mention of the new service version.

File: sdk/storage/azure-storage-file-share/HISTORY.md [hunk 3]
```diff
@@ -1,6 +1,16 @@
 # Release History
 
-## 12.0.1 (Unreleased)
+## 12.1.0
+
+**New features**
+- Added support for the 2019-07-07 service version.
+- Added support for file leases. Including adding both sync and async `ShareFileLeaseClient`, accepting leaseIds for operations that support leases.
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-file-share/HISTORY.md]: `ShareFileLeaseClient` was introduced to both sync and async versions of the SDK, which allows users to perform operations on file leases.

File: sdk/storage/azure-storage-file-share/HISTORY.md [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
+- Added `failed_handles_count` to `close_handle` and `close_all_handles` returned dict to allow users to access number of failed handles.
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-file-share/HISTORY.md]: Fix sentence.

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Pull request merged**